src/sync/configure.rs

```diff
@@ -31,7 +31,10 @@ pub fn configure_tool(tool_name: &str, config_asset: &ConfigAsset) -> Tool {
 fn full_configure(config_asset: &ConfigAsset) -> Option<ToolInfo> {
     let owner = config_asset.owner.clone()?;
     let repo = config_asset.repo.clone()?;
-    let exe_name = config_asset.exe_name.clone()?;
+    let exe_name = config_asset
+        .exe_name
+        .clone()
+        .unwrap_or(config_asset.repo.clone()?);
```

Comment:
`config_asset.repo.clone()?` is already evaluated to `repo` variable (that has type `String`) from the previous line so this can be simplified:
```suggestion
        .unwrap_or(repo);
```